scripts/run_lighthouse_tests.py

```diff
@@ -190,29 +206,31 @@ def main(args=None):
     """Runs lighthouse checks and deletes reports."""
     parsed_args = _PARSER.parse_args(args=args)
 
-    if not (parsed_args.accessibility or parsed_args.performance):
-        _PARSER.error(
-            'Must pass either --accessibility or --performance flag')
+    if parsed_args.mode == 'accessibility':
+        lighthouse_mode = LIGHTHOUSE_MODE_ACCESSIBILITY
+        server_mode = SERVER_MODE_DEV
+    elif parsed_args.mode == 'performance':
+        lighthouse_mode = LIGHTHOUSE_MODE_PERFORMANCE
+        server_mode = SERVER_MODE_PROD
 
     enable_webpages()
     atexit.register(cleanup)
-    performance_mode = parsed_args.performance
 
-    if performance_mode:
+    if lighthouse_mode == LIGHTHOUSE_MODE_PERFORMANCE:
```

Comment:
Ditto -- use "try"/"except KeyError".

Make sure to add tests for when an invalid param is passed in.

Reply:
Sorry -- scratch the first part. Do "else raise" instead.